atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryAtlasDbFactory.java

```diff
@@ -82,7 +82,15 @@ public String getType() {
         return "memory";
     }
 
-    // async initialization not implemented/propagated
+    /**
+     * Creates an InMemoryKeyValueService.
```

Comment:
Again, we could use a note explicitly calling out that async initialisation is not supported.